practice-one/src/components/footer.html

```diff
@@ -0,0 +1,44 @@
+<footer class="footer-container">
+  <div class="website-summary-container">
+    <div class="brief-description">
+      <h3 class="brief-heading">Foodhut</h3>
+      <p class="brief-content">Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor </p>
+      <div class="brief-icon-container">
+        <div class="facebook-icon"><img src="./assets/images/facebook.svg" alt="Facebook icon"></div>
+        <div class="instagram-icon"><img src="./assets/images/instagram.svg" alt="Instagram icon"></div>
+        <div class="twitter-icon"><img src="./assets/images/twitter.svg" alt="Twitter icon"></div>
+      </div>
+    </div>
+
+    <div class="about-us-infomation-container">
+      <h3 class="about-us-heading">About Us</h3>
+      <ul class="about-us-list">
+        <li class="about-us-item"><a href="">About Us</a></li>
+        <li class="about-us-item"><a href="">Service Us</a></li>
+        <li class="about-us-item"><a href="">Contact</a></li>
+        <li class="about-us-item"><a href="">Company</a></li>
+      </ul>
+    </div>
+
+    <div class="company-information-container">
+      <h3 class="company-information-heading">Company</h3>
+      <ul class="company-information-list">
+        <li class="company-information-item"><a href="">Partnership</a></li>
+        <li class="company-information-item"><a href="">Terms of Use</a></li>
+        <li class="company-information-item"><a href="">Privacy</a></li>
+        <li class="company-information-item"><a href="">Sitemap</a></li>
+      </ul>
+    </div>
+
+    <form class="subscribe-form" action="URL" method="post">
```

Comment:
`action="URL" method="post"` what this for

Reply:
it is unecessary in this case, so i deleted it, Thanks for your contribution